checkov/terraform/checks/resource/kubernetes/HostPort.py

```diff
@@ -9,7 +9,7 @@ def __init__(self):
         """
                https://kubernetes.io/docs/concepts/configuration/overview/
 
-               Don’t specify a hostPort for a Pod unless it is absolutely necessary.
+               Don't specify a hostPort for a Pod unless it is absolutely necessary.
```

Comment:
This line was changed by the script I made to go over this repo to find bad unicode characters.
I left it in the PR just because unicode seems unnecessary here. There's no bug in this line.